src/test/python_tests/test_code_actions.py

```diff
@@ -20,8 +20,11 @@
     ("code", "contents", "command"),
     [
         (
-            "C0301:line-too-long",
-            "FRUIT = ['apricot', 'blackcurrant', 'cantaloupe', 'dragon fruit', 'elderberry', 'fig', 'grapefruit']",  # # pylint: disable=line-too-long
+            "C0301:line-too-long", "C0305:trailing-newlines",
+            # # pylint: disable=line-too-long
+            "FRUIT = ['apricot', 'blackcurrant', 'cantaloupe', 'dragon fruit', 'elderberry', 'fig', 'grapefruit']",
+            # # pylint: disable=trailing-newlines
+            "VEGGIE = ['carrot\n', 'radish\n', 'cucumber\n', 'potato\n', '\n']",
```

Comment:
The changes did not look syntactically right. The suggested code might help.

```suggestion
            "C0301:line-too-long", "C0305:trailing-newlines",
            # # pylint: disable=line-too-long
            "FRUIT = ['apricot', 'blackcurrant', 'cantaloupe', 'dragon fruit', 'elderberry', 'fig', 'grapefruit']",
            {
                "title": f"{LINTER}: Run document formatting",
                "command": "editor.action.formatDocument",
                "arguments": None,
            },
        ),
        (
            # # pylint: disable=trailing-newlines
            "VEGGIE = ['carrot\n', 'radish\n', 'cucumber\n', 'potato\n', '\n']",
```